docs/apache-airflow/faq.rst

```diff
@@ -119,6 +119,34 @@ How do I trigger tasks based on another task's failure?
 
 You can achieve this with :ref:`concepts:trigger-rules`.
 
+How to control DAG file parsing timeout for different DAG files?
+----------------------------------------------------------------
+(only valid for Airflow >= 2.3.0)
```

Comment:
```suggestion

(only valid for Airflow >= 2.3.0)
```

Reply:
```suggestion

 .. versionadded:: 2.3
 
```

Reply:
@ashb looks like we used the format of `(only valid for Airflow` in this doc.

let me know your thoughts. thanks

Reply:
There's only one other use -- your call.